pb-rs/src/types.rs

```diff
@@ -312,7 +312,7 @@ impl FieldType {
                 format!("{}{}{}", m.get_modules(desc), m.name, lifetime)
             }
             FieldType::Map(ref key, ref value) => format!(
-                "HashMap<{}, {}>",
+                "KVMap<{}, {}>",
```

Comment:
I believe most people won't use no_std and they'd rather use regular `std::HashMap`.
I understand that the distinction is done at import time but I'd like things to be as simple as possible for std users.